api/v1alpha2/template.go

```diff
@@ -71,6 +69,18 @@ type Action struct {
 // See https://docs.docker.com/storage/volumes/ for additional details.
 type Volume string
 
+// Namespace defines the Linux namespaces to use for the container.
```

Comment:
Based on the type name and the field above in the Actions I assume this is Namespaces :)